netpyne_ui/netpyne_geppetto.py

```diff
@@ -342,7 +342,15 @@ def getAvailableCellTypes(self):
             if ct not in cellTypes:
                 cellTypes.add(ct)
         return cellTypes
+    
+    def getAvailableSections(self):
```

Comment:
If you just do:
```
sections = []
for cellRule in netParams.cellParams:
  sections += netpyne_geppetto.netParams.cellParams[cellRule]['secs'].keys()
return sections
```

I think it is much simpler, right? Anyway, have a look at my review here https://github.com/MetaCell/geppetto-netpyne/pull/45#pullrequestreview-135394508. You may need to refactor this method because of my comments

Reply:
I don't want to repeat items in the list. But I could do as you say and then a `list(set(sections))` in the return